### PR TITLE
Updated Copyright and URL

### PR DESCRIPTION
**Pull request by jjasghar**
- It's 2016 now, so...updated that
- Our main site is chef.io now, not getchef.com

### CODE REVIEW

File: omnibus/files/private-chef-cookbooks/private-chef/files/default/html/404.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <body>
   <div class="header-block">
     <div id="header">
-      <strong class="logo"><a href="http://www.getchef.com">Chef</a></strong>
+      <strong class="logo"><a href="http://www.chef.io">Chef</a></strong>
```
Comment (smith) [inline omnibus/files/private-chef-cookbooks/private-chef/files/default/html/404.html]: Might as well make it https and save them a redirect.

Comment (jjasghar) [inline omnibus/files/private-chef-cookbooks/private-chef/files/default/html/404.html]: Will do, the other one wasn't so I'll change them both.

**Comment by stevendanna** [comment]
:+1:

**Comment by jjasghar** [comment]
I don't think the failing test: https://travis-ci.org/chef/chef-server/jobs/114904254 is apropos here.

**Comment by tylercloke** [comment]
@jjasghar fix merged to master. Rebase and you should be good!

**Pull request merged**